contributing-tips.rst

```diff
@@ -0,0 +1,174 @@
+Tips for Making Good Contributions
+====================================
+
+.. _small-pr:
+
+Smallest meaningful PR
+------------------------
+
+A PR should normally address one issue. This makes it easier to review, easier to deploy, and easier to roll back in case of a problem. Additionally, the smaller the PR, the less likely it is to create a merge issue.
+
+The exception is when several issues are closely related or can reasonably be worked on together.  In this case, it should be clear by looking at the conversation on the Github issues that the items are related and will be worked on together. Your PR message should also make it clear which issues are being worked on, and whether the PR closes the issues or not.
```

Comment:
Adding an example of issue reference like "Closes https://github.com/opendatakit/docs/issues/{no}" will be a good idea I guess. 
We can frame it this way :
Each PR description must end with "Closes https://github.com/opendatakit/docs/issues/{no}" if it closes the issue or "Addresses https://github.com/opendatakit/docs/issues/{no}" if there's work remaining.

Reply:
Also I don't know um maybe we could have a standard format for a pull request? For example: 
PR title : It should explain what the PR is addressing and not how, and why. The character limit could be decided.
PR description : A short 4-5 lines paragraph briefly explaining how it has been done.
Issue reference : Closes/Addresses/Fixes {url} 
where url is the complete url of issue being worked on.